### DISCUSSION TITLE
Change from nullable to optional

### DISCUSSION THREAD

**Discussion by Schmell** [chrishoermann/zod-prisma-types#182] [Q&A] [answered]
```prisma
model MyModel {
  id             String        @id @unique @default(cuid())
  name           String
  description    String?
}
```
generates
```javascript
export const MyModelSchema = z.object({
  id: z.string().cuid(),
  name: z.string(),
  description: z.string().nullable(),
})
```
for the `description` field i need it to be `.optional()`  instead of `.nullable()` for svelte [superForms ](https://superforms.rocks/) to mark the fields as not required
I tried to use string validator, but no luck and i don't really want to add a hundred of comments to my schema anyway
Thanks in advance!

**Comment by chrishoermann** [comment, marked as answer]
can you try 
```prisma
generator zod {
  // ...rest of config
  writeNullishInModelTypes = true
}
```
as mentioned in [the docs](https://github.com/chrishoermann/zod-prisma-types#writenullishinmodeltypes) and see if `.nullish()` is sufficient?

**Reply by Schmell** [reply]
I saw that option, but didn't think it would work.
Thanks a bunch
